pkg/virt-controller/watch/migration.go

```diff
@@ -729,11 +737,54 @@ func (c *MigrationController) handleTargetPodHandoff(migration *virtv1.VirtualMa
 	return nil
 }
 
-func (c *MigrationController) handleSignalMigrationAbort(migration *virtv1.VirtualMachineInstanceMigration, vmi *virtv1.VirtualMachineInstance) error {
+func (c *MigrationController) failMigration(isAborted bool, migration *virtv1.VirtualMachineInstanceMigration, vmi *virtv1.VirtualMachineInstance, pod *k8sv1.Pod) {
```

Comment:
i don't see why we need this helper function when we have many other places in the code that sets `phase = failed` without using this helper. It's confusing. Why does the helper even exist to begin with if it's only kind of used here and there.

If you just need a helper function to set a condition in the situation where an abort occured, then make a helper that sets that condition.

Reply:
:+1: